crates/ra_db/src/input.rs

```diff
@@ -1,10 +1,11 @@
-/// This module specifies the input to rust-analyzer. In some sense, this is
-/// **the** most important module, because all other fancy stuff is strictly
-/// derived from this input.
-///
-/// Note that neither this module, nor any other part of the analyzer's core do
-/// actual IO. See `vfs` and `project_model` in the `ra_lsp_server` crate for how
-/// actual IO is done and lowered to input.
+//! This module specifies the input to rust-analyzer. In some sense, this is
+//! **the** most important module, because all other fancy stuff is strictly
+//! derived from this input.
+//!
+//! Note that neither this module, nor any other part of the analyzer's core do
+//! actual IO. See `vfs` and `project_model` in the `ra_lsp_server` crate for how
+//! actual IO is done and lowered to input.
```

Comment:
:laughing: yeah, confusing `///` and `//!` is also a good reason to have such check